bot/seasons/evergreen/nationday.py

```diff
@@ -0,0 +1,149 @@
+import datetime
+import json
+import logging
+from pathlib import Path
+from typing import List, Tuple
+
+import discord
+import requests
+from discord.ext.commands import Bot, Cog, Context, command
+
+from bot.pagination import ImagePaginator
+
+
+logger = logging.getLogger(__name__)
+# RESTful API to get countries info
+url = "https://restcountries.eu/rest/v2/alpha/"
+# white flag image to show during error
+white_flag = "https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/\
+White_flag_waving.svg/158px-White_flag_waving.svg.png"
+# url to get images of flags
+flag_url = "https://www.countryflags.io/"
```

Comment:
This could be a template:
```python
flag_url = "https://www.countryflags.io/{country_code}/flat/64.png"
....
flag_url.format(country_code="DE")  # German flag
```
For your consideration: could there be a case where we want to parametrize the style and size also?

Reply:
Well, 64 is the biggest size, and appears reasonably descriptive on an embed. So I'm not sure about size. As for style, there's only one other style, which is shiny... which I don't think will really make a difference to the final outcome.